lib/experimental/Navigation/ApplicationFrame/index.tsx

```diff
@@ -19,57 +19,74 @@ export function ApplicationFrame({ children, sidebar }: ApplicationFrameProps) {
   )
 }
 
+const SkipToContentButton = ({ contentId }: { contentId?: string }) => {
+  return (
+    <a
+      href={`#${contentId}`}
+      className={focusRing(
+        "absolute z-50 -translate-y-full translate-x-4 bg-f1-background-secondary-hover p-2 text-lg no-underline transition-transform duration-200 focus-visible:translate-y-0"
+      )}
+    >
+      Skip to content
+    </a>
+  )
+}
+
 function ApplicationFrameContent({ children, sidebar }: ApplicationFrameProps) {
   const { sidebarState, toggleSidebar, isSmallScreen } = useSidebar()
   const shouldReduceMotion = useReducedMotion()
 
   return (
-    <MotionConfig
-      reducedMotion={shouldReduceMotion ? "always" : "never"}
-      transition={{
-        ease: [0.25, 0.1, 0.25, 1],
-        duration: shouldReduceMotion ? 0 : 0.2,
-      }}
-    >
-      <div className="relative flex h-full flex-row">
-        <AnimatePresence>
-          {sidebarState === "unlocked" && (
-            <motion.div
-              className={cn("fixed inset-0 z-[5] bg-f1-background-bold/60", {
-                hidden: !isSmallScreen,
-              })}
-              initial={{ opacity: 0 }}
-              animate={{ opacity: 1 }}
-              exit={{ opacity: 0 }}
-              transition={{ duration: shouldReduceMotion ? 0 : 0.2 }}
-              onClick={toggleSidebar}
-            />
-          )}
-        </AnimatePresence>
-        <div
-          className={cn(
-            { "transition-all": !shouldReduceMotion },
-            sidebarState === "locked" ? "w-64 pl-3" : "w-0"
-          )}
-        >
-          {sidebar}
+    <>
+      <SkipToContentButton contentId="content" />
```

Comment:
The fact that you need to know the `contentId` here and we're shifting the responsibility to the user makes me think this belongs to the `ApplicationFrame` and not the sidebar.

Reply:
Sir, you are a gentleman and a scholar.